sway/criteria.c

```diff
@@ -373,3 +373,25 @@ list_t *criteria_for(swayc_t *cont) {
 	}
 	return matches;
 }
+
+void container_for(swayc_t *container, list_t *tokens, list_t *list) {
```

Comment:
I wanted to use container map, but that only takes two arguments when I would need three: tokens, container and final list. I could solve that by passing a (container list,tokens list) struct or use a global, but both felt somewhat jurry-rigged.